bin/avif.js

```diff
@@ -18,9 +21,17 @@ const { glob } = require("tinyglobby");
 const convert = require("../lib/convert");
 
 const avif = async () => {
-  const files = await glob([input], { absolute: true });
+  const prompt = `${input.replace(/\\/g, "/")}/**/*.{${target}}`; // Include all files in subdirectories
+
+  const files = await glob([prompt], {
+    absolute: true,
+    onlyFiles: true,
+    deep: recursive ? Infinity : 0, // Adjust depth based on recursive option
```

Comment:
Is `deep` a real option? I can see `maxDepth` mentioned in the [readme](https://github.com/isaacs/node-glob#options).

Reply:
Sorry, ignore this, I was looking at `glob` rather than `tinyglobby`.